libs/generic-view/ui/src/lib/generated/mc-contacts-view.ts

```diff
@@ -1150,6 +1150,47 @@ export const generateMcContactsView: ComponentGenerator<McContactsView> = (
         ],
       },
     },
+    contactDetailsPhoneNumber1DefaultIconWrapper: {
+      component: "block-plain",
+      childrenKeys: ["contactDetailsPhoneNumber1DefaultIconTooltip"],
+    },
+    contactDetailsPhoneNumber1DefaultIconTooltip: {
+      component: "tooltip",
+      config: {
+        offset: {
+          x: 15,
+          y: 15,
+        },
+        placement: "bottom-right",
+      },
+      childrenKeys: [
+        "contactDetailsPhoneNumber1DefaultIconTooltipAnchor",
+        "contactDetailsPhoneNumber1DefaultIconTooltipContent",
+      ],
+    },
+    contactDetailsPhoneNumber1DefaultIconTooltipAnchor: {
+      component: "tooltip.anchor",
+      childrenKeys: ["contactDetailsPhoneNumber1DefaultIcon"],
+    },
+    contactDetailsPhoneNumber1DefaultIconTooltipContent: {
+      component: "tooltip.content",
+      childrenKeys: [
+        "contactDetailsPhoneNumber1DefaultIconTooltipContentTextWrapper",
+      ],
+    },
+    contactDetailsPhoneNumber1DefaultIconTooltipContentTextWrapper: {
+      component: "p5-component",
+      config: {
+        color: "grey1",
+      },
+      childrenKeys: ["contactDetailsPhoneNumber1DefaultIconTooltipContentText"],
+    },
+    contactDetailsPhoneNumber1DefaultIconTooltipContentText: {
+      component: "format-message",
+      config: {
+        messageTemplate: "Calls and SMS will go to this number by default.",
+      },
+    },
```

Comment:
This can be simplified
```suggestion
    contactDetailsPhoneNumber1DefaultIconTooltipContentTextWrapper: {
      component: "p5-component",
      config: {
        text: "Calls and SMS will go to this number by default."
        color: "grey1",
      },
    },
```